### ISSUE TITLE
Bug bounty and testing tracker

### ISSUE THREAD

**Issue by dexX7** [OmniLayer/omnicore#216]
To quote the [annoucement on omnilayer.org](http://www.omnilayer.org/#!testing/c17xy):

> **Crowdsourced Testing!**

> You can add value to this protocol today for the small price of 15 minutes. In return, the Omni Foundation will pay 10 OMNI for each replicatible bug you can find. We'll also pay 5 OMNI to all well-formed alternative test cases we haven't thought of yet. UI issues and confusing flow also qualifies, we'll post those on a list of new issues. Qualitative issues people report on will be paid from a pot of 250 OMNI.

> Help us launch the Decentralized Exchange! The more people test, the faster we can launch. This bounty is open until September 20th or until the bounty is exhausted. 

This thread shall serve to list and discuss findings, and I will try to keep this post updated.

Please let me know, if I missed something, especially @zathras-crypto, because there were often issue discussions in past, which were not strictly posted in seperate issues.

If you consider something that is listed here not as qualified, please leave a comment, and notes regarding the unconfirmed or dismissed issues are also very welcome!

Disclaimer: I'm not the one who ultimately pays out the bounties, and the listing here should provide an overview, but not imply any guarantees. (CC: @patrickdugan)

---

Confirmed issues:
```
#142  zathras-crypto   Noticed issue with PR 142 ("omni_decodetransaction")
#251  dexX7            Granted user amount out of bounds when reaching crowdsale limit
#247  dexX7            Potential overflow in crowdsale participations (+ #248, #249)
#246  zathras-crypto   Crowdsale participation transactions missing in RPC result
#242  zathras-crypto   Reorg containing STO corrupts STODB data
#236  dexX7            Raised amount returned by "omni_getcrowdsale" is always zero
#235  achamely         Bug: Crowdsale Tx's being classified as simple sends via rpc
#222  via email        Preview build fails consensus checkpoint on Windows
#219  zathras-crypto   Bitcoin balance disappears from the UI
#215  nanomo           Assertion failed after run first time Windows 10
#212  achamely         Bug: getsto_MP / omni_getsto <hash> * doesn't work
#206  dexX7            MetaDEx cancellation combo box does not update after cancel
#200  dexX7            Remaining amount of MetaDEx trade is not updated
```
Not necessarily bugs, or unconfirmed issues:
```
#208  dexX7            Missing orders in MetaDEx trade history
#199  achamely         Omnicore ping/socket timeout | frozen ?
```

---

Three quick questions:

- we may label qualified issues, which is probably easier to track (yes? no? label? milestone?)
- should we leave a commend in each qualified issue, to give the submitter a heads up? (yes? no?)
- how are "winners" notified? (we could ping them via GitHub, but I'm not sure, if this is reliable)

**Comment by dexX7** [comment]
@patrickdugan notified me to close this issue, as the contest is over.

Patrick, please post further instructions to redeem the bounties.

@achamely, @nanomo, @zathras-crypto: gratulations!

**Comment by nanomo** [comment]
yay!

**Comment by patrickdugan** [comment]
Thanks for all this Dexx.

Have you been able to resolve these issues yet? Do they have separate issue histories? 

Please post addresses and Craig will pay out the appropriate amounts within a week or so, he has the keys but is very busy, if would be best if people could submit their addresses before the meeting at 5pm EST tomorrow.

**Comment by dexX7** [comment]
You're welcome!

Yes, the listed issues were actual issues, and we fixed or addressed them. :)

All fixes, except the one for https://github.com/OmniLayer/omnicore/pull/142#issuecomment-144600367 are also already merged into the main branch, whereby this one is fixed, but not yet merged.

You can see details for each case by appending the number in the first row to the URL, like this:

- `https://github.com/OmniLayer/omnicore/issues/[number]` (example: https://github.com/OmniLayer/omnicore/issues/215)

---

@achamely and @zathras-crypto are likely attending to the meeting.

Please note that #222 was initially reported via email by Neil at CrowdBucks.com.

Ping @nanomo.

Here is mine: 1FALZeJNm8C7rhejQRVzN7XtGJsymWZ3iz

**Comment by nanomo** [comment]
Here is mine, thanks a lot!
1BQMgxdS7cHZFQoXJ5ENCAUzTNipFGTEuQ

**Comment by zathras-crypto** [comment]
Here is mine, thanks!
1omniZNVq9FCHLjDDLoGM3eTvm1mkDc26

**Comment by achamely** [comment]
And mine
1BjpKchLXJPBssSVz7B8f8owXpFPjT1Aen

**Comment by zathras-crypto** [comment]
Paging @patrickdugan - can we please take care of nanomo/dexx/adam in terms of bounties?  Thanks :)

**Comment by CraigSellars** [comment]
Just waiting on transfer of Dev MSC / Dev OMNI from JR and we’ll be distributing shortly thereafter.

**Comment by dexX7** [comment]
@achamely, @nanomo, @zathras-crypto:

The bounties were sent. Please see the following spreadsheet for payout details and transasctions:

https://docs.google.com/spreadsheets/d/17Rj_8XZpcJhpIMFlivzZzOQo1FhtM4WvCcwBf34ZGT4/
